### ISSUE TITLE
Windows firewall blocks VirtualBox

### ISSUE THREAD

**Issue by achekulaev** [docksal/docksal#204]
Sometimes firewall blocks VirtualBox. Folder mounting fails like this:
```
mount: mounting //192.168.64.1/c on /c failed: Operation now in progress
```

We need to automatically create rule for windows firewall which manually can be created in the following way:

```
Click on inbound rules (left column), then on new rule (right column). Choose custom rule, set the rule to allow all programs, and any protocol. For the scope, add in the first box (local IP addresses) 192.168.64.1, and in the second box (remote IP) 192.168.64.100. Click next, select allow the connection, next, check all profiles, next, give it a name and save.
```

**Comment by ComboPrime** [comment]
Have tried adding the above rule, but still getting this error on two Windows boxes that previously were able to run Docksal (I think corporate changed our security settings recently):

`mount: mounting //192.168.64.1/c on /c failed: Host is down
exit status 255
Error creating share`

**Comment by achekulaev** [comment]
Yeah there was some security change on Windows recently that threw a wrench into Docksal's ability to mount folders in many different ways. We're still in the middle of identifying the root of change and how to address it.

**Comment by ComboPrime** [comment]
Thanks for the quick response!

Good luck,

etg

**Comment by ComboPrime** [comment]
Hi folks—just wondering if there's any status update on this issue. Thanks for all your hard work.

**Comment by lmakarov** [comment]
@ComboPrime we are not sure that the firewall is causing the issues when mounting Windows shares. Take a look at #117 and the troubleshooting guide were started here: 
https://github.com/docksal/docksal/blob/develop/docs/troubleshooting-smb.md

Please report in #117 your results.

**Comment by achekulaev** [comment]
There is new release that might address many windows issues. I would recommend updating. I would recommend restarting the *host* machine after update.
